src/service/holdings.ts

```diff
@@ -154,6 +178,12 @@ export async function getHoldingsOther() {
       cMC02Balance(),
     ])
   )
+  /*TODO:
+  Uncomment and check once assets are bridged and pools are active
+  ethHeld.value += await uniV3HoldingsForToken(RESERVE_MULTISIG_CELO, ETH_AXELAR_ADDRESS)
+  ethHeld.value += await uniV3HoldingsForToken(RESERVE_MULTISIG_CELO, ETH_WORMHOLE_ADDRESS)
+  btcHeld.value += await uniV3HoldingsForToken(RESERVE_MULTISIG_CELO, BTC_AXELAR_ADDRESS)
+  btcHeld.value += await uniV3HoldingsForToken(RESERVE_MULTISIG_CELO, BTC_WORMHOLE_ADDRESS)*/
```

Comment:
Is there any reason to not merge with these uncommented? It looks like they should work even when there's no balance yet.

Reply:
Check with Olek to ask for addresses and lets make this script final and have it work. Also I think we already have a stCelo to Celo pool with a bit of liquidity (~10k Celo total) on that multisig, and I know he's also bridging the rest now.